Name: Hide Maps (Pre3)
Version: 2.2.4-1
Author: webOS Archive
Description: Hide Maps icon from Launcher on Pre3

--- .orig/media/cryptofs/apps/usr/palm/applications/com.palm.app.maps/appinfo.json
+++ /media/cryptofs/apps/usr/palm/applications/com.palm.app.maps/appinfo.json
@@ -8,7 +8,8 @@
   "version": "3.1.32",
   "type": "web",
   "uiRevision": 2,
-  "removable": false,
+  "removable": true,
+  "visible": false,
   "main": "index.html",
   "vendor": "HP"
 }
\ No newline at end of file
